.infra/infra/main.tf

```diff
@@ -0,0 +1,270 @@
+// SNOWFLAKE DATABASE
```

Comment:
If we also want to use this for demo, we'll need to add the other stuff as well: 

https://demo.raito.dev/data-objects/FxI9V-eipHJ4jgTNnjUpx/general

Also some more tagging may be useful to demo ABAC (like data_category, department, ...): to be discussed

Reply:
I agree. I focussed on the tests we have right now. We can of course discuss (maybe together with Bart) on how that should look like.
Ideally we have the same dataset for testing as for our demo setup.

Reply:
Bart told me that the current data for demo is good (it has a bit of everything: sales, ordering, HR, ...) which is good preparing different demos